### DISCUSSION TITLE
For Left-handers: Tray to the Right

### DISCUSSION THREAD

**Discussion by Aithie** [zsviczian/obsidian-excalidraw-plugin#475] [Ideas]
1) Would it be possible for the tray to offer an option for left-handers i.e move to the right side of the screen? 

2) Would it be possible for the tool selection bar to move to the right? 

I am thinking in terms of using the iPad with the pencil and no keyboard being attached so you can use your right hand to select things on the far right side of the screen while using the pencil with your left hand.

Thank you for all your amazing work!

**Comment by zsviczian** [comment]
I solved this in 1.6.18. Now there is a setting to move the tray to the right.

**Reply by Aithie** [reply]
This is amazing! Thanks!
